browser_wrapper.go

```diff
@@ -4,47 +4,50 @@ import (
 	"log"
 
 	"github.com/playwright-community/playwright-go"
+	"go.k6.io/k6/js/modules"
 )
 
 type browserWrapper struct {
 	Browser playwright.Browser
+	vu      modules.VU
 }
 
-func newBrowserWrapper(browser playwright.Browser) *browserWrapper {
+func newBrowserWrapper(browser playwright.Browser, vu modules.VU) *browserWrapper {
 	return &browserWrapper{
 		Browser: browser,
+		vu:      vu,
 	}
 }
 
 func (b *browserWrapper) Close() {
 	if err := b.Browser.Close(); err != nil {
-		log.Fatalf("could not close browser: %v", err)
+		Throw(b.vu.Runtime(), "Error closing browser", err)
 	}
 }
 
 func (b *browserWrapper) NewPage() *pageWrapper {
 	page, err := b.Browser.NewPage()
 
 	if err != nil {
-		log.Fatalf("could not create new page: %v", err)
+		Throw(b.vu.Runtime(), "Error creating new page", err)
 	}
 
-	return newPageWrapper(page)
+	return newPageWrapper(page, b.vu)
 }
 
 // Cookies wrapper around playwright cookies fetch function
 func (b *browserWrapper) Cookies() []*playwright.BrowserContextCookiesResult {
 	if b.Browser == nil {
-		log.Fatalf("looks like there's no browser attached. please call `launch()` before accessing the cookies.")
+		log.Panicf("It looks like there's no browser attached. please call `launch()` before accessing the cookies.")
```

Comment:
Shouldn't we be using `Throw` here as well? and also at line [45](https://github.com/LendingHome/xk6-playwright/pull/15/files#diff-2620d226100044ee08f0fdb3fa5d5e946602fb2e9ce65cce3d96b64b4fe3330cR45)